File: code/ch09/9.1.1.two_sum.py
class Solution:
    def twoSum(self, numbers:List[int], target:int)->List[int]:
        for i in range(0, len(numbers), 1):
            for j in range(i + 1, len(numbers), 1):
                if numbers[i] + numbers[j] == target:
                    return [i + 1, j + 1]

        return []

File: code/ch09/9.1.1.two_sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        vector<int> ret;
        for (int i = 0; i <= numbers.size() - 1 - 1; i++) {
            for (int j = i + 1; j <= numbers.size() - 1; j++) {
                if (numbers[i] + numbers[j] == target) {
                    ret.push_back(i + 1);
                    ret.push_back(j + 1);
                }
            }
        }
        return ret;
    }
};
